public/index.html

```diff
@@ -23,12 +26,13 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React + MDB by Christian Aichner</title>
+    <title>Marstime</title>
   </head>
   <body>
     <noscript>
-      You need to enable JavaScript to run this app.
+      You need to enable JavaScript to rüun this app.
     </noscript>
+
```

Comment:
You can remove this line

Reply:
You should really start to use suggestions.
It is generally a cool feature. xD